coordinator/src/app.ts

```diff
@@ -1,20 +1,6 @@
-import {Boom} from '@hapi/boom'
 import Hapi from '@hapi/hapi'
 import routes from './routes'
 
-const CONTENT_TYPE = 'application/fhir+json; fhirVersion=4.0'
-
-const preResponse = function (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) {
-    const response = request.response
-    if (response instanceof Boom) {
-        console.log(response)
```

Comment:
Will we see errors in logs if we remove this line?

Reply:
Good point